### PR TITLE
Feature/288 submit review at any time

### PR DESCRIPTION
**Pull request by todd-dembrey**

### CODE REVIEW

File: opentech/apply/review/models.py [hunk 1]
```diff
@@ -100,24 +101,16 @@ def __str__(self):
...
     @property
-    def fields(self):
-        return [
-            field.render(context={'data': data})
-            for data, field in self.data_and_fields()
-        ]
-
-    def render_answers(self):
-        return mark_safe(''.join(self.fields))
+    def for_latest(self):
```
Comment (zerolab) [inline opentech/apply/review/models.py]: make this more explicit? i.e. `for_latest_revision`

File: opentech/apply/review/models.py [hunk 2]
```diff
@@ -100,24 +101,16 @@ def __str__(self):
...
+        return self.revision == self.submission.live_revision
+
+    def get_compare_url(self):
```
Comment (zerolab) [inline opentech/apply/review/models.py]: This is another one of those cross-over moments.
Can this be renamed to be more verbose? again, this is the submission compare url, not the review one and it reads   as the latter

File: opentech/apply/review/templates/review/review_detail.html [hunk 3]
```diff
@@ -18,9 +18,17 @@ <h5>Recommendation</h5>
         <h5>Score</h5>
         <p>{{ review.score }}</p>
     </div>
+    {% if not review.for_latest %}
+        <div>
+            <h5>Review was not against the lasted version:</h5>
```
Comment (zerolab) [inline opentech/apply/review/templates/review/review_detail.html]: "The Review was not added against the latest submission version"?
(or at least `s/lasted/latest`)

File: opentech/apply/review/views.py [hunk 4]
```diff
@@ -130,25 +129,28 @@ def get_context_data(self, **kwargs):
...
-
-                    if isinstance(field.block, ScoreFieldBlock):
-                        value = json.loads(data)
-                        rating_value = int(value[1])
-                        rating = RATE_CHOICES_DICT.get(rating_value, RATE_CHOICE_NA)
-                        comment = str(value[0])
-                        review_data[field.id]['answers'].append(rating + comment)
-                    else:
-                        review_data[field.id]['answers'].append(str(data))
+                    # If we haven't see the question before we backfill with '-'
```
Comment (zerolab) [inline opentech/apply/review/views.py]: backfilling with ''. no?

Comment (todd-dembrey) [reply opentech/apply/review/views.py]: and that is why i dont like comments..

Comment (zerolab) [reply opentech/apply/review/views.py]: :)

File: opentech/apply/review/templates/review/review_detail.html [hunk 5]
```diff
@@ -20,7 +20,7 @@ <h5>Score</h5>
     </div>
     {% if not review.for_latest %}
         <div>
-            <h5>Review was not against the lasted version:</h5>
+            <h5>Review was not against the lastest version:</h5>
```
Comment (zerolab) [inline opentech/apply/review/templates/review/review_detail.html]: "latest" ;)

**Pull request merged**